xbmc/video/dialogs/GUIDialogVideoManager.h

```diff
@@ -28,6 +28,7 @@ class CGUIDialogVideoManager : public CGUIDialog
   virtual void SetVideoAsset(const std::shared_ptr<CFileItem>& item);
   virtual void SetSelectedVideoAsset(const std::shared_ptr<CFileItem>& asset);
   virtual bool HasUpdatedItems() const { return m_hasUpdatedItems; }
+  virtual void RefreshSelectedVideoAsset();
```

Comment:
Why is this `virtual`? I would make it only virtual if there is actual a reason. Virtual calls are more expensive than normal function calls (vtable lookup).

Reply:
Not really useful today, it can easily be changed later if we need a different behavior for versions and extras.
It doesn't have to be public either. I pushed an updated.

Reply:
unrelated: shouldn't the destructor be virtual though?

Reply:
> unrelated: shouldn't the destructor be virtual though?

It is. => https://github.com/xbmc/xbmc/blob/master/xbmc/video/dialogs/GUIDialogVideoManager.h#L26C3-L26C48